code/logistic_sgd.py

```diff
@@ -433,5 +438,37 @@ def sgd_optimization_mnist(learning_rate=0.13, n_epochs=1000,
                           os.path.split(__file__)[1] +
                           ' ran for %.1fs' % ((end_time - start_time)))
 
+
+def predict():
+    """
+    An example of how to load a trained model and use it
+    to predict labels.
+    """
+
+    # load the saved model
+    classifier = cPickle.load(open('best_model.pkl'))
+    y_pred = classifier.y_pred
+
+    # find the input to theano graph
+    inputs = graph.inputs([y_pred])
```

Comment:
I think we told we don't use that as this is for the DLT. In the `__init__`, keep a reference to input. Do that for each example to be consistent, but you don't need to add a predict() function for each example